src/dotnet/commands/dotnet-update/tool/LocalizableStrings.resx

```diff
@@ -0,0 +1,183 @@
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="PackageIdArgumentName" xml:space="preserve">
+    <value>PACKAGE_ID</value>
+  </data>
+  <data name="PackageIdArgumentDescription" xml:space="preserve">
+    <value>NuGet Package Id of the tool to update.</value>
+  </data>
+  <data name="SpecifyExactlyOnePackageId" xml:space="preserve">
+    <value>Please specify one tool Package Id to update.</value>
+  </data>
+  <data name="VersionOptionDescription" xml:space="preserve">
+    <value>Version of the tool package in NuGet.</value>
+  </data>
+  <data name="SourceOptionDescription" xml:space="preserve">
+    <value>Specifies a NuGet package source to use during update.</value>
+  </data>
+  <data name="SourceOptionName" xml:space="preserve">
+    <value>SOURCE</value>
+  </data>
+  <data name="CommandDescription" xml:space="preserve">
+    <value>Updates a tool for use on the command line.</value>
+  </data>
+  <data name="ConfigFileOptionDescription" xml:space="preserve">
+    <value>The NuGet configuration file to use.</value>
+  </data>
+  <data name="FrameworkOptionDescription" xml:space="preserve">
+    <value>The target framework to update the tool for.</value>
+  </data>
+  <data name="NuGetConfigurationFileDoesNotExist" xml:space="preserve">
+    <value>NuGet configuration file '{0}' does not exist.</value>
+  </data>
+  <data name="UpdateSucceeded" xml:space="preserve">
+    <value>Tool '{0}' (from version '{1}' to version'{2}') was successfully updated.</value>
+  </data>
+  <data name="GlobalOptionDescription" xml:space="preserve">
+    <value>Update user wide.</value>
+  </data>
+  <data name="UpdateFullCommandNameLocalized" xml:space="preserve">
+    <value>.NET update Command</value>
+  </data>
+  <data name="ToolAlreadyInstalled" xml:space="preserve">
+    <value>Tool '{0}' is already installed.</value>
+  </data>
+  <data name="UpdateToolCommandNeedGlobalOrToolPath" xml:space="preserve">
+    <value>Please specify either the global option (--global) or the tool path option (--tool-path).</value>
+  </data>
+  <data name="UpdateToolCommandInvalidGlobalAndToolPath" xml:space="preserve">
+    <value>(--global) conflicts with the tool path option (--tool-path). Please specify only one of the options.</value>
+  </data>
+  <data name="ToolPathDescription" xml:space="preserve">
+    <value>Location of shim to access tool</value>
+  </data>
+  <data name="ToolHasMultipleVersionsInstalled" xml:space="preserve">
+    <value>Tool '{0}' has multiple versions installed and cannot be uninstalled.</value>
+  </data>
+  <data name="ToolNotInstalled" xml:space="preserve">
+    <value>Tool '{0}' is not currently installed.</value>
+  </data>
+  <data name="UpdateSucceededVersionNoChange" xml:space="preserve">
+    <value>Tool '{0}' was successfully updated with no version change.</value>
```

Comment:
Perhaps this should be `Tool '{0}' (version '{1}') is already the latest stable version.` or something?  A message implying that it was updated but the version didn't change is confusing, at least to me.

Reply:
I am not sure about it as well. But it could be a different feed. So it could be a different package with the same version. And we actually did a uninstall install, something might changed, at least files timestamp.

Reply:
I actually think this is going to be super common until we get a method to check for a new update, do we really want to uninstall and reinstall in this case? Assuming no change to that, I would like a friendlier message. Maybe:

Tool '{0}' was reinstalled with the latest stable version (version '{1}')

Reply:
@KathleenDollard what do you think about this issue?

Reply:
I'm good with Kathleen's string.  One thing we didn't consider, though, is that `update` can function as a downgrade when the user has a higher versioned preview package installed.  It should probably call out in the help somewhere that this upgrades to the latest **stable** version.

Reply:
So the description should be `Updates a tool to the latest stable version for use.`( used to be `Updates a tool for use on the command line.`and Kathleen suggested to drop on the command line in another comment)

Reply:
Changed as suggested and also changed the command description to "Updates a tool to the latest stable version for use."